### PR TITLE
Added new tool: sigthief

### PR DESCRIPTION
**Pull request by mrsmith0x00**
Stealing Signatures and Making One Invalid Signature at a Time.

### CODE REVIEW

File: packages/sigthief/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='sigthief'
```
Comment (FFY00) [inline packages/sigthief/PKGBUILD]: No need for quotes. This might actually be a bad thing.

File: packages/sigthief/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+arch=('any')
+url='https://github.com/secretsquirrel/SigThief'
+license=('BSD')
+depends=('python')
+makedepends=('git')
+source=('git+https://github.com/secretsquirrel/SigThief.git')
+sha1sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/SigThief"
```
Comment (FFY00) [inline packages/sigthief/PKGBUILD]: No need to cd to `$srcdir`, we already start there.

Just:
```
cd SigThief
```

File: packages/sigthief/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd "$srcdir/SigThief"
```
Comment (FFY00) [inline packages/sigthief/PKGBUILD]: No need to cd to `$srcdir`, we already start there.

Just:
```
cd SigThief
```

File: packages/sigthief/PKGBUILD (hunk 1 above)
Comment (noptrix) [reply packages/sigthief/PKGBUILD]: i will change it

**Pull request merged**